compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangParserListener.java

```diff
@@ -2648,6 +2651,24 @@ public void exitSimpleLiteral(BallerinaParser.SimpleLiteralContext ctx) {
         } else if ((node = ctx.QuotedStringLiteral()) != null) {
             String text = node.getText();
             text = text.substring(1, text.length() - 1);
```

Comment:
I think it is better to merge these two lines in this PR as well - 

`String text = node.getText().substring(1, text.length() - 1);`

Reply:
We cannot do that because we need to get `text.length()`